### ISSUE TITLE
Formatted Error Shows Raw HTML

### ISSUE THREAD

**Issue by joshua-lafever** [yoavbls/pretty-ts-errors#52]
**Describe the bug**
The error formatting is odd and is displaying what appears to be an icon that isn't properly interpolated. 

This is the line that caused the error:

<code>return (control.value > max) ? { maxDate: true  } | null;</code>

The error is being caused by accidentally typing `|` instead of `:`, but the formatting of the error is strange.

I haven't seen any other instances like this and I've been using this extension for the past week.

**Expected behavior**
To show the styled error.

**Original error**
<code>The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.</code>

**Screenshots**

![image](https://user-images.githubusercontent.com/51811500/235257952-9dbdbad4-a5e7-41e6-8e3f-91b891e4aa95.png)

**Comment by Sheraff** [comment]
I got the exact same error and came to report it as well. Here's mine:

```ts
latinReleases.sort((a, b) => a["release-group"]["first-release-date"] - b["release-group"]["first-release-date"])
```

No syntax error (just a type coercion that Typescript doesn't like). Here's the output:

<img width="629" alt="Screen Shot 2023-04-29 at 18 00 09" src="https://user-images.githubusercontent.com/1325721/235312034-aefe25d7-6db2-4e2c-a317-68c062ce3096.png">

I'm on v0.3.0, with vscode 1.77.3

**Comment by yoavbls** [comment]
It was fixed in [v0.4.0](https://github.com/yoavbls/pretty-ts-errors/releases/tag/v0.4.0), take a look 🙂
